src/database/migrations/1731482774441-create-accommodation-and-amenity-and-category.ts

```diff
@@ -0,0 +1,63 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class CreateAccommodatonAndAmenityAndCategory1731482774441
```

Comment:
I think you forgot to revert your previous migration and generate a new one. Now your migration does not create an accommodation and amenity table.

Reply:
how can I generate new migration for three tables? Accommodation , Amenity and Category